index.html

```diff
@@ -125,55 +224,60 @@ <h2 class="Tonic">Multy Post <br>Stories</h2>
+                        <li><a href="#"><img src="icons/Vector (3).png" alt="Chill_icon"></a></li>
+                    </ul>
+                </div> 
+                <button type="button" id="seeProject" class="seeproject">Get my resume</button>   
+            </div>
+            <div id="about">
+                <ul class="languages">
                     <li><h4>Languages</h4></li>
-                    <li><img src="icons/un.png" alt="un.png"></li>
+                    <li><img class="separ" src="icons/un.png" alt="un.png"></li>
                 </ul>
-            </li>
-            <li class="prlanguages">
-                <ul>
+                <ul class="prlanguages" id="prlanguages">
                     <li><img src="icons/Ellipse 1.png" alt="Ellipse">Javascript</li>
                     <li><img src="icons/Ellipse 1 (1).png" alt="Ellipse">HTML</li>
                     <li><img src="icons/Ellipse 1 (2).png" alt="Ellipse">CSS</li>
                 </ul>
-            </li>
-            <li class="languages">
-                <ul class="skillss">
+                <ul class="languages">
                     <li><h4>Frameworks</h4></li>
-                    <li><img src="icons/u.png" alt="un.png"></li>
+                    <li><img class="separ" src="icons/u.png" alt="un.png"></li>
                 </ul>
-            </li>
-            <li><img class="separa" src="icons/Separator Bottom.png" alt="Separator"></li>
-            <li class="languages">
-                <ul class="skillss">
+                <img class="separa" src="icons/Separator Bottom.png" alt="Separator">
+                <ul class="languages">
                     <li><h4>Skills</h4></li>
-                    <li><img src="icons/u.png" alt="un.png"></li>
+                    <li><img class="separ" src="icons/u.png" alt="un.png"></li>
                 </ul>
-            </li>
-            <li><img class="separa" src="icons/Separator Bottom.png" alt="Separator"></li>
-        </ul>
-    </section>  
+                <img class="separa" src="icons/Separator Bottom.png" alt="Separator">
+            </div>
+        </section> 
+        <section id="contactform" class="contactform">
+            <article class="contactme" id="contactme">
+                <h2 class="cme">Contact me</h2>
+                <p class="pme">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </article>
+            <form class="form" action="https://formspree.io/f/mnqydvzj" method="post">
+                <input name="Name" id="name" type="text" placeholder="Name..." maxlength="30" required><br>
+                <input name="Email" id="email" type="email" placeholder="you@example.com" required><br>
+                <textarea name="message" id="textarea" placeholder="Write your message here..." maxlength="500" required></textarea><br>
```

Comment:
**OPTIONAL** - The gap between your input element is too much and does not make your project look like the design template. Please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
**OPTIONAL** - Also, the background image in this section looks distorted. Please fix this 🙏 

Your project:
![thsethset](https://user-images.githubusercontent.com/73607512/237027323-9148727f-0c57-413c-a25b-8774ca397f82.PNG)